poet.go

```diff
@@ -11,24 +11,36 @@ import (
 
 	"github.com/jessevdk/go-flags"
 	"github.com/spacemeshos/smutil/log"
-)
 
-var cfg *config
+	"github.com/spacemeshos/poet/config"
+	"github.com/spacemeshos/poet/server"
+)
 
 // poetMain is the true entry point for poet. This function is required since
 // defers created in the top-level scope of a main method aren't executed if
 // os.Exit() is called.
 func poetMain() error {
```

Comment:
I think it is weird that `poetMain()` is in the `config` package.

Reply:
It's in the `main` package.